site/using-cluster-operator.md

```diff
@@ -0,0 +1,790 @@
+# Using Kubernetes Cluster Operator
+
+This section covers how to deploy Custom Resource objects that will
+be managed by the Kubernetes Cluster Operator. If you have not installed
+the Cluster Operator, check the [Installing Cluster Operator in a Kubernetes cluster](/install-cluster-operator.html).
+
+## Confirm Service Availability
+
+Before configuring your app to use Cluster Operator, ensure that RabbitmqCluster Custom Resource is deployed
+to your Kubernetes cluster and is available.
+
+To confirm this availability:
+
+1. Run:
+
+    ```
```

Comment:
I'm afraid our site does not support GFMD "code fences". You have to use `<pre class="lang-{language}">` blocks that Highlight.js will recognise.